modules/hc_resource/config/install/core.entity_form_display.node.hc_resource.default.yml

```diff
@@ -5,7 +5,6 @@ dependencies:
     - entity_browser.browser.hc_media_browser_file
     - field.field.node.hc_resource.body
     - field.field.node.hc_resource.field_hc_header_image
-    - field.field.node.hc_resource.field_hc_hide_header_image
```

Comment:
I dont fully understand why this was removed, don't we need this field on there?